src/run.ts

```diff
@@ -32,11 +38,10 @@ function getHelmDownloadURL(version: string): string {
         case 'Windows_NT':
         default:
             return util.format('https://get.helm.sh/helm-%s-windows-amd64.zip', version);
-
     }
 }
 
-async function getStableHelmVersion(): Promise<string> {
+export async function getStableHelmVersion(): Promise<string> {
```

Comment:
Why export?

Reply:
I actually copied from the latest master. @sundargs2000 has recently added tests which need the functions to be exported. I'll probably have to add exports to other functions as well to resolve merge conflicts.